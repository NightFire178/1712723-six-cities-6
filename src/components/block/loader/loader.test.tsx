import  React from 'react'
import {render} from '@testing-library/react'
import Loader from './loader'

test(`loader component block test`, ()=>{
  const {container} = render(<Loader />)
  expect(container).toMatchSnapshot()
})
