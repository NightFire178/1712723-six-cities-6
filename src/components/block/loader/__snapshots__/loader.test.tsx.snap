// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loader component block test 1`] = `
<div>
  <div
    class="cssload-loader-block"
  >
    <div
      class="cssload-loader"
    >
      <div
        class="cssload-inner cssload-one"
      />
      <div
        class="cssload-inner cssload-two"
      />
      <div
        class="cssload-inner cssload-three"
      />
    </div>
  </div>
</div>
`;
